stdlib/ballerina-http/src/main/ballerina/http/websocket/websocket_client.bal

```diff
@@ -69,17 +62,16 @@ public type WebSocketClient object {
         return conn;
     }
 
-    documentation {
-        Starts the registered service.
-    }
-    public native function start();
-
     documentation {
         Stops the registered service.
     }
     public function stop() {
         WebSocketConnector webSocketConnector = getCallerActions();
-        error|() ignoredValue = webSocketConnector.close(1001, "The connection has been stopped");
+        error|() value = webSocketConnector.close(1001, "The connection has been stopped");
```

Comment:
It is better to use 1000 instead of 1001. 
Look at the [status codes](https://tools.ietf.org/html/rfc6455#section-7.4) in the spec. Also, the reason text is wrong. IMO it is better to use a standard text from the spec rather than putting something on our own. Such as `normal closure` for `1000` or `Going away` for `1001`.